runtime/queries/metricsview.go

```diff
@@ -322,17 +334,29 @@ func convertToXLSXValue(pbvalue *structpb.Value) (interface{}, error) {
 
 func metricsViewDimensionToSafeColumn(mv *runtimev1.MetricsViewSpec, dimName string) (string, error) {
 	dimName = strings.ToLower(dimName)
+	dimension, err := findDimension(mv, dimName)
+	if err != nil {
+		return "", err
+	}
+	return safeName(getMetricsViewDimensionName(dimension)), nil
+}
+
+func findDimension(mv *runtimev1.MetricsViewSpec, dimName string) (*runtimev1.MetricsViewSpec_DimensionV2, error) {
```

Comment:
For consistency with other functions here, consider calling the func `metricsViewDimension(mv, name)` (it's a big package with many util funcs)